codec-http/src/main/java/io/netty/handler/codec/http/CombinedHttpHeaders.java

```diff
@@ -213,9 +232,13 @@ public CombinedHttpHeadersImpl setObject(CharSequence name, Iterable<?> values)
             return this;
         }
 
+        private boolean canBeCombined(CharSequence name) {
```

Comment:
You always use the opposite `!canBeCombined` so IMHO it would be better to have a `cantBeCombined` instead (I would expect IntelliJ to complain as well).

Reply:
Oh yeah good point, at some point I was using `canBeCombined` without the negation and wanted to avoid the double negative but it's gone now, so will change it.